sec2_the_ruby_programming_language/2.9_authenticator.rb

```diff
@@ -0,0 +1,39 @@
+users = [
+          {username: "mashrur", password: "password1"},
```

Comment:
???

Reply:
Should I change the tab space to 2 in this case? Because I followed the indentation shown in the video for this one.

Reply:
I told you about consistency. If you going to use 2 tab, then make sure EVERYTHING is 2 tab. not some 2 tab, some 2 space.
Make a choice and stick with it.

Reply:
Alright sure noted thankss